theHarvester.py

```diff
@@ -30,7 +30,17 @@
 
 Core.banner()
 
-
+def modified_soruce(engines, excluded_engines):
+    engines = Core.get_supportedengines()
+    engines.remove('all')
+    excluded_engines = set(map(str.strip, excluded_engines.split(',')))
+    for exclude in excluded_engines:
```

Comment:
Instead of doing this for loop you can simply do engines = engines.difference(excluded_engines). This will subtract the two sets and is much cleaner. 
Ex: engines= {'Google', 'duckduckgo', 'baidu'}
      excluded_engines = {'baidu'}
      print(engines.difference(excluded_engines))
      {'Google','duckduckgo'}

Reply:
Updated thanks!